vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Locale;
+import java.util.Objects;
+
+import com.vaadin.flow.component.AbstractField;
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.HasSize;
+import com.vaadin.flow.component.HasStyle;
+import com.vaadin.flow.component.HasValue;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.dependency.HtmlImport;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.component.timepicker.TimePicker;
+
+/**
+ * Server-side component that encapsulates the functionality of the
+ * {@code vaadin-date-time-picker} web component.
+ *
+ */
+@Tag("vaadin-date-time-picker")
+@HtmlImport("frontend://bower_components/vaadin-date-time-picker/src/vaadin-date-time-picker.html")
+@NpmPackage(value = "@vaadin/vaadin-date-time-picker", version = "1.0.0-alpha1")
+@JsModule("@vaadin/vaadin-date-time-picker/src/vaadin-date-time-picker.js")
+public class DateTimePicker extends AbstractField<DateTimePicker, LocalDateTime>
+        implements HasStyle, HasSize {
+
+    private final DatePicker datePicker = new DatePicker();
+    private final TimePicker timePicker = new TimePicker();
+    private Locale locale;
+
+    /**
+     * Default constructor.
+     */
+    public DateTimePicker() {
+        this((LocalDateTime) null);
+    }
+
+    /**
+     * Convenience constructor to create a date time picker with a pre-selected
+     * date in current UI locale format.
```

Comment:
Constructor docs still have "date" instead of "date time"

Reply:
Fixed.

Reply:
2 more constructors. Sorry if I wasn't explicit enough that there are multiple of those.

Reply:
💩

Reply:
Fixed. This time I checked with search for "date" to see that all instances should be followed by "time" in that file.